.vscode/launch.json

```diff
@@ -9,7 +9,7 @@
             "type": "python",
             "request": "launch",
             "program": "${workspaceFolder}/.vscode/check.py",
-            "pythonPath": "${workspaceFolder}/.venv/bin/python",
+            "python": "${workspaceFolder}/.venv/bin/python",
```

Comment:
I would avoid changes in this file for our fork.

Reply:
I _think_ that Andrea changed this because he is using vscode and the keyword changed? If this change is to make vscode work, I think this is fine.

Reply:
Yes, I didn't changed manually, my vscode did it.